app_customers/lib/main.dart

```diff
@@ -82,9 +85,21 @@ class _BuildApp extends StatelessWidget {
       ],
       supportedLocales: AppInternationalization.supportedLocales,
       title: AppEnvironment.appName,
-      translations: AppInternationalization(Get.deviceLocale ?? Locale('en')),
+      translations: localization,
       locale: Get.deviceLocale,
       fallbackLocale: Locale('en', ''),
+      builder: (context, child) {
```

Comment:
Make it here will be displayed on all our pages; we don't want it. we have our custom scaffold widget  https://github.com/karibu-cap/trans_all_client/blob/bafee67f8bb8ae849311465a9aa7c1dd7a769070/app_customers/lib/widgets/custom_scaffold.dart#L9